docker-compose.yml

```diff
@@ -34,18 +34,38 @@ services:
     networks:
       - microservices
 
+  postgres-auth:
```

Comment:
Do you really need two database servers? You can create just two databases inside same postgres server